### PR TITLE
Missing check

### PR DESCRIPTION
**Pull request by Tinkaa**
Add check for missing dates and for missing population data. 
If missing dates, add them and give warning. If population is 0, set to NaN and give warning. 

Main changes are in the .py files, so would be great if you could check them. 
Plus possibly the rounding of triggers in the .ipynb of `ethiopia/IPC_ThresholdAnalysis.ipynb`

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -29,7 +31,7 @@ MWI:
   fewsnet_dates: [ "200907","200910","201001","201004","201007","201010","201101","201104","201107","201110","201201",
                    "201204","201210","201301","201304","201307","201310","201401","201404","201407","201410","201501","201504",
                    "201507","201510","201602","201606","201610","201702","201706","201710","201802","201806","201812",
-                   "201902","201906","201910","202002","202006","202008" ]
+                   "201902","201906","201910","202002","202006","202008","202010"]
```
Comment (joseepoirier) [inline config.yml:34]: How do you envision keeping this config list up-to-date and accurate? Should we pre-populate the scheduled publications for the next year or two? How would we know that a date is "missing"?

Comment (Tinkaa) [reply config.yml:34]: Those are very good points, that need some dev thought. I added issues for them for now and will try to get back to those issues soonish.

**Comment by Tinkaa** [comment]
@joseepoirier same here, do you think this code is good to go for merge, given that the issues you brought up will be thought of later? Feel free to disagree

**Comment by joseepoirier** [comment]
@Tinkaa Yes I think we can merge for now. I'll leave it with you to follow-up on the process questions. Thanks!

**Pull request merged**